Formula/ghostscript.rb

```diff
@@ -49,7 +49,6 @@ def install
       --disable-gtk
       --disable-fontconfig
       --without-libidn
-      --with-system-libtiff
       --without-x
```

Comment:
It looks like this is no longer a valid option and is now the default (there is an option for `--with-x `), so I think this line can be removed.

Reply:
Done.

Reply:
Just tried the recipe locally. Removing `--without-x` gives me the following error.
```
In file included from ./devices/gdevxalt.c:21:
./base/x_.h:131:12: fatal error: 'X11/Xlib.h' file not found
#  include <X11/Xlib.h>
           ^~~~~~~~~~~~
1 error generated.
make: *** [obj/gdevxalt.o] Error 1
```

Reply:
Ah, ok -- please disregard my suggestion then,  sorry about that.

Reply:
I just added `--without-x` back.

Reply:
Thank you, sorry for the trouble.